Assets/Scripts/Tank/Player/TankController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using UnityEngine;
+
+[RequireComponent(typeof(Animator), typeof(Rigidbody))]
+public class TankController : MonoBehaviour, Idamagable
+{
+    private float horizontalMove;
+    private float verticalMove;
+
+    private Rigidbody playerBody;
+
+    [SerializeField]
+    private float speed = 0.8f;
+
+    [SerializeField]
+    private float thrust = 0.8f;
+
+    [SerializeField]
+    private int health = 100;
+    private Joystick joystick;
+    public Joystick Joystick{
+        set{
+            joystick = value;
+        }
+    }
+
+    #region Movement Physics
```

Comment:
What if Start implements functionality other than Physics? Does it still make sense to keep it inside the region then?